src/api-endpoints.ts

```diff
@@ -1,14 +1,22 @@
-import { ISdkParams } from './http-client';
+import { BASE_URL as DEFAULT_BASE_URL, UFC_ENDPOINT } from './constants';
+import { IQueryParams } from './http-client';
 
-const UFC_ENDPOINT = '/flag-config/v1/config';
+interface IApiEndpointsParams {
+  queryParams: IQueryParams;
```

Comment:
looks like it's optional so you can make it `queryParams?: IQueryParams;`

Reply:
Sounds good. Made it optional

Reply:
thanks for iterating on it 🙏🏽